subprojects/docs/src/docs/release/notes.md

```diff
@@ -124,6 +124,13 @@ includeBuild("../project-with-plugin-and-library")
 This distinction reflects what Gradle offers for repository declarations - 
 repositories are specified separately for plugin dependencies and for production dependencies.
 
+### Support for plugins with multiple variants
```

Comment:
Does the plugin portal already support GMM? If it doesn't, we should note somewhere that the multiple variants don't work for plugins published to the portal. Or do we have the work for fixing the portal already lined up?

Reply:
Yes it is planed: #15992 

I added a note to the issue to update the doc/notes in the unlikely case that it is not done.